requirements.txt

```diff
@@ -1,9 +1,3 @@
-ipython==4.0.2
-nose
-mock
 requests
-pandas==0.17.1
```

Comment:
Why were these removed?  As far as I can tell, this repo still depends on:

* ipython
* nose
* mock
* pandas
* ipykernel
* ipywidgets